integration_tests/steps/storage_test.py

```diff
@@ -104,16 +104,14 @@ def check_capacity_dependant_sell_rate(context):
 def check_custom_storage(context):
     house1 = list(filter(lambda x: x.name == "House 1", context.simulation.area.children))[0]
     storage = list(filter(lambda x: x.name == "H1 Storage1", house1.children))[0]
-    price_dec_per_slot = 0.1 * int(context.simulation.simulation_config.slot_length.seconds /
-                                   context.simulation.simulation_config.tick_length.seconds)
     trades_sold = []
+    break_even_sell = list(storage.strategy.break_even.values())[-1][-1]
```

Comment:
Can you explain a bit why is the last value of the break_even profile needed here?

Reply:
Technically all the break_even_sell rate are same, so I chose the last one. Changing it to get break_even_sell at that market_slot

Reply:
Thanks, that will make it less confusing for the reader.